### PR TITLE
🖋: Add developer guide

### PR DESCRIPTION
**Pull request by in-a-paddy**
## やったこと

- [x] Add developer guide (Hugo)
- [x] Move documents from README to docs

## やっていないこと

Nothing

## 動作確認

- [x] `hugo serve`

## デバイス

It's nothing about the devices.

## その他（レビュアーへの申し送り事項、懸念点など）

Nothing

### CODE REVIEW

File: docs/content/react-native/_index.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (Takumon) [inline docs/content/react-native/_index.md:1]: The background color of the header is transparent in this Hugo theme.
When I scroll through the document, The header text overlaps with the page text as shown below.

image

**Review by Takumon — COMMENTED** [review]
Please check a comment.

File: docs/content/react-native/_index.md (hunk 1 above)
Comment (in-a-paddy) [reply docs/content/react-native/_index.md:1]: Configured theme styles 👍

Comment (Takumon) [reply docs/content/react-native/_index.md:1]: Awesome 😎

**Review by Takumon — APPROVED** [review]
LGTM✨

**Pull request merged**